client/admin-pointers.js

```diff
@@ -37,15 +37,21 @@ import jQuery from 'jquery';
 
 				show_pointer( pointers, i + 1 );
 			},
-		} );
+			open: function() {
+				if ( !pointer.dim ) {
+					return;
+				}
 
-		function open() {
-			target.pointer( options ).pointer( 'open' );
-			if ( pointer.dim ) {
```

Comment:
Moved this `if ( pointer.dim )` logic to the `onOpen` event on the pointer - this way the`wcs-pointer-page-dimmer` is added _only_ when the pointer has also been added.

This is safe to work with the PayPal integration - the PP integration does not use the dimmer